### ISSUE TITLE
LookupError: Unknown property 'buffered_writes'  - only when running peakrdl regblock

### ISSUE THREAD

**Issue by Peter-McLean-Intel** [SystemRDL/PeakRDL#10]
When I run
```
python my_regblock.py
```
The contents of which are very simple (based on the intro page of peakrdl-regblock documentation), the above works and outputs my files.

```
peakrdl regblock system.rdl -o output_dir -cpuif axi4-lite-flat
```

I get the following exception:

```
Traceback (most recent call last):
  File "/PeakRDL-regblock/venv/bin/peakrdl", line 33, in <module>
    sys.exit(load_entry_point('peakrdl==0.3.0', 'console_scripts', 'peakrdl')())
  File "/PeakRDL-regblock/venv/lib/python3.10/site-packages/peakrdl/main.py", line 70, in main
    options.subcommand.main(options)
  File "/PeakRDL-regblock/venv/lib/python3.10/site-packages/peakrdl/subcommand.py", line 123, in main
    self.do_export(top, options)
  File "/PeakRDL-regblock/venv/lib/python3.10/site-packages/peakrdl/plugins/exporter.py", line 63, in do_export
    func(top_node, options)
  File "/PeakRDL-regblock/venv/lib/python3.10/site-packages/peakrdl_regblock/__peakrdl__.py", line 94, in do_export
    x.export(
  File "/PeakRDL-regblock/venv/lib/python3.10/site-packages/peakrdl_regblock/exporter.py", line 134, in export
    scanner.do_scan()
  File "/PeakRDL-regblock/venv/lib/python3.10/site-packages/peakrdl_regblock/scan_design.py", line 64, in do_scan
    RDLWalker().walk(self.exp.top_node, self)
  File "/PeakRDL-regblock/venv/lib/python3.10/site-packages/systemrdl/walker.py", line 159, in walk
    self.walk(child, *listeners)
  File "/PeakRDL-regblock/venv/lib/python3.10/site-packages/systemrdl/walker.py", line 150, in walk
    self.current_action = self.do_enter(node, listener)
  File "/PeakRDL-regblock/venv/lib/python3.10/site-packages/systemrdl/walker.py", line 193, in do_enter
    new_action = listener.enter_Reg(node) or WalkerAction.Continue
  File "/PeakRDL-regblock/venv/lib/python3.10/site-packages/peakrdl_regblock/scan_design.py", line 93, in enter_Reg
    self.has_buffered_write_regs = self.has_buffered_write_regs or bool(node.get_property('buffered_writes'))
  File "/PeakRDL-regblock/venv/lib/python3.10/site-packages/systemrdl/node.py", line 409, in get_property
    raise LookupError("Unknown property '%s'" % prop_name)
LookupError: Unknown property 'buffered_writes'
```

```
peakrdl --version
0.3.0
cat venv/lib64/python3.10/site-packages/peakrdl_regblock/__about__.py 
__version__ = "0.9.0"
```

**Comment by amykyta3** [comment]
The regblock exporter now supports some RDL extensions via user-defined properties.
When creating the exporter instance, there is one new step that has been added where these UDPs are pre-registered with the compiler. See the updated example code here: https://peakrdl-regblock.readthedocs.io/en/latest/#quick-start

Specifically:
``` python
from peakrdl_regblock.udps import ALL_UDPS
...
# Register all UDPs that 'regblock' requires
for udp in ALL_UDPS:
    rdlc.register_udp(udp)
```

What this does is register several "soft" UDPs into the back-end that suppresses the lookup error you are encountering. To be clear, these UDPs are _not_ loaded into the SystemRDL namespace, they are only pre-registered into the compiler in order to make development of these extensions easier. You can read more about this here: https://systemrdl-compiler.readthedocs.io/en/stable/api/udp.html

Regarding the PeakRDL command line tool - that is my mistake. I implemented the mechanism to pre-load the UDPs automatically, but forgot to release it. v0.4.0 will be published in a few minutes!

Keep in mind that the PeakRDL tool and PeakRDL-regblock use v0.x versions which mean they are in pre-production beta, so there still may be a few quirks as I figure out how I want the tools to work. Thanks for being an early adopter and helping figure out issues!

**Comment by amykyta3** [comment]
v0.4.0 has been released.
Let me know if you have further questions

**Comment by Peter-McLean-Intel** [comment]
This is fixed in v0.4.0.
